index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Resume Builder</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700,800&display=swap" rel="stylesheet"> 
+</head>
+<body>
+    <header>
+        <!-- Header -->
+    </header>
+    <main class="wrapper">
+        <div class="profile section">
+            <div class="title">
+                <h4 class="profileTag">Profile</h4>
+            </div>
+            <!-- Profile -->
+        </div>
+        <hr>
+        <div class="skills section">
+            <div class="title">
+                <h4 class="skillsTag">Skills</h4>
+            </div>
+            <!-- Skills -->
+        </div>
+        <hr>
+        <div class="education section">
+            <div class="title">
+                <h4 class="educationTag">Education</h4>
+            </div>
+            <div class="educationDiv">
+            </div>
+                <!-- Education -->
+        </div>
+        <hr>
+        <div class="projects section">
+            <div class="title">
+                <h4 class="projectsTag">Past Projects</h4>
+            </div>
+            <div class="context">
+            <div class="proj">
+                <div class="projInfo">
+                    <p class="projTitle">Podcast Commuter</p>
+                    <p class="projDesc">Use Podcast Commuter to help you pick a podcast based on your length of commute.</p>
+                    <a href="http://juliadufresne.com/podcastcommuter/">View Live</a>
```

Comment:
Potentially could have this data included with the resume JSON. Same with the other past project(s) below.
But the index.html overall is really well-written!